include/revng/Yield/PTML.h

```diff
@@ -67,4 +68,41 @@ functionLink(std::string_view Location, const model::Binary &Binary);
 std::string
 shallowFunctionLink(std::string_view Location, const model::Binary &Binary);
 
+/// Emits PTML containing a comment for a function constructed based on
+/// the model representation of both the function and its type.
+///
+/// \param Function the function to gather comment information from.
+/// \param Binary the model of the binary the function belongs to.
+/// \param CommentIndicator the indicator used at the start of commented lines.
+///        If no custom value is specified, the value obtained from
+///        `model::Architecture::getAssemblyCommentIndicator` is used instead,
+///        which is the preferred option for the assembly (see the inline
+///        overload calling this function).
+///        For c code, the recommended indicator is either `'///'` or `'  *'`.
+///        With second option, the opening (`'/**'`) and closing (`' */'`)
+///        brackets are not provided by this function and need to be added
+///        by the caller.
+/// \param LineWidth the expected width of a line within the emitted comment.
+/// \param Indentation the number of spaces appended **before** the comment as
+///        an indentation token (see `ptml::tokens::Indentation`).
+///
+/// \returns a serialized PTML string containing the comment.
+std::string functionComment(const model::Function &Function,
+                            const model::Binary &Binary,
+                            std::string_view CommentIndicator,
+                            std::size_t Indentation = 0,
+                            std::size_t LineWidth = 80);
```

Comment:
I'd drop the default values for `CommentIndicator`, `Indentation` and `LineWidth`, let the caller be explicit.
If needed, declare a global variable `DefaultIndetnation`.

Reply:
Either way works for me.

Reply:
Done.